src/lib/theme/defaultTheme.ts

```diff
@@ -307,20 +307,20 @@ export default {
    */
 
   card: {
-    base: `
-      bg-white dark:bg-scaleDark-700
```

Comment:
ah, it was because it should of been `dark:bg-scale-300` not `dark:bg-scaleDark-700`

Reply:
yes